src/components/AdmissionsDash.js

```diff
@@ -85,6 +87,30 @@ export class AdmissionsDash extends React.Component {
     this.setState({ data: newData }, function () {});
   };
 
+  // ConnectMerakiApi = () => {
```

Comment:
@Poonam-Singh-Bagh Remove unused codes. Leave comments only if they are used to document something

Reply:
@Poonam-Singh-Bagh